src/js/actions/guides.js

```diff
@@ -313,6 +313,36 @@ define(function (require, exports) {
     setGuide.transfers = [resetGuidePolicies, deleteGuide];
 
     /**
+     * Clears all the guides in the given document
+     *
+     * @param {{id: number}} document Document model or an object containing document ID
+     * @return {Promise}
+     */
+    var clearGuides = function (document) {
+        if (document === undefined) {
```

Comment:
this is purely a style nit, but seems like this could simplified to doc = doc || this.flux....getCurrentDOcument()